protx/decorators.py

```diff
@@ -3,33 +3,49 @@
 from diskcache import Cache
 import os
 import logging
-from flask import request
+from flask import request, redirect, url_for
 import requests
 
 logger = logging.getLogger(__name__)
 
 cache = Cache("database_cache")
 
+def is_setup_complete(function, redirect_onboarding, *args, **kwargs):
```

Comment:
to simplify things could `is_setup_complete` just return a true or false  `def is_setup_complete() -> bool: `